ftl/README.md

```diff
@@ -20,6 +20,27 @@ This command can be read as "Build the source code in directory `$dir` into an i
 
 These binaries **do not depend on Docker**, and construct images directly in the registry.
 
+So for example will we demonstrate using the Node FTL builder to create a container for a node application from a node app's source code:
+Assume we are deploying the node source for the app https://github.com/JustinBeckwith/cloudcats.  First downlowad the node ftl.par file from https://storage.googleapis.com/gcp-container-tools/ftl/node/latest/ftl.par.  Then we run the ftl.par file pointing to our application:
+```shell
+$ ftl.par --directory=$HOME/cloudcats/web --base=gcr.io/google-appengine/nodejs:latest --image=gcr.i
+o/my-project/cloudcats-node-app:latest
+```
+
+## Releases
+Currently FTL is released in .par format for each supported runtime.
+node:
+[https://storage.googleapis.com/gcp-container-tools/ftl/node/latest/ftl.par]()
```

Comment:
Looks like you meant to do something like 

```
[FTL node par](https://storage.googleapis.com/gcp-container-tools/ftl/node/latest/ftl.par)
```
for these links.

Reply:
changed the format here